tests/testImuPreintegration.cpp

```diff
@@ -76,7 +76,7 @@ TEST(TestImuPreintegration, LoadedSimulationData) {
   imuPreintegratedParams->gyroscopeCovariance = I_3x3 * pow(gyrNoiseSigma, 2);
   imuPreintegratedParams->biasOmegaCovariance = I_3x3 * pow(gyrBiasRwSigma, 2);
   imuPreintegratedParams->integrationCovariance = I_3x3 * integrationCovariance;
-  imuPreintegratedParams->biasAccOmegaInt = I_6x6 * biasAccOmegaInt;
+  imuPreintegratedParams->biasAccOmegaInit = I_6x6 * biasAccOmegaInit;
```

Comment:
if not there yet, I suggest adding a unit test on the block-wise implementation of G * cov * G'

Reply:
This will have to be an MCMC test similar to ImuFactor.